packages/ndla-ui/src/RadioButtonGroup/component.radio-button-group.scss

```diff
@@ -42,6 +42,7 @@
   &__input {
     opacity: 0;
     position: absolute;
+    width: auto;
```

Comment:
Denne rare bredden på input-feltet var pga `input` har satt `width: 100%` i editorial, som gir mening for text-inputs (siden de skal ta opp hele containeren), men brekker radio inputs (Kunne også potensielt løses med klasse definert styling e.l i ed, men syns denne `auto` saken er mindre ekkel enn `!important`).